src/client/components/Avatar/Avatar.css

```diff
@@ -0,0 +1,52 @@
+.avatar-student-comp {
+  height: 316.9661865234375px;
```

Comment:
please use rem or em, not px ... because of the responsivness
how did you calculate all px?

Reply:
Probably c/p from Figma.